lib/Entry.hs

```diff
@@ -76,11 +79,25 @@ instance A.FromJSON FieldVisibility where
     "private" -> pure Private
     other -> fail $ "expecting either 'public' or 'private', but found: '" <> T.unpack other <> "'"
 
+newtype FieldValue = FieldValue { fvValue :: T.Text }
```

Comment:
According to the [style guide](https://github.com/serokell/coffer/blob/main/docs/code-style.md#records), a newtype's field accessor should be named `un<newtype name>` (in this case, `unFieldValue`)

Reply:
After that fix, the `value` lens won't be generated.

Reply:
It will generate a `fieldValue` lens instead.

If you add this at the top of the module, you can see the code generated by TH:

```hs
{-# OPTIONS_GHC -ddump-splices -dsuppress-uniques #-}

newtype FieldValue = FieldValue { unFieldValue :: T.Text }
  deriving stock (Show, Eq, Ord)
makeLensesWith abbreviatedFields ''FieldValue
```

```
/home/dc/dev/serokell/coffer/lib/Entry.hs:87:1-45: Splicing declarations
    makeLensesWith abbreviatedFields ''FieldValue
  ======>
    class HasFieldValue s a | s -> a where
      fieldValue :: Lens' s a
    instance HasFieldValue FieldValue Text where
      {-# INLINE fieldValue #-}
      fieldValue = (iso (\ (FieldValue x) -> x)) FieldValue
```


Even better, since this is a newtype, we could use `makeLensesFor` to generate an `Iso` instead of a `Lens`.

```hs
makeLensesFor [("unFieldValue", "fieldValue")] ''FieldValue
```
```
/home/dc/dev/serokell/coffer/lib/Entry.hs:88:1-59: Splicing declarations
    makeLensesFor [("unFieldValue", "fieldValue")] ''FieldValue
  ======>
    fieldValue :: Iso' FieldValue Text
    fieldValue = (iso (\ (FieldValue x) -> x)) FieldValue
    {-# INLINE fieldValue #-}
```

Reply:
Done